cdc/owner/schema.go

```diff
@@ -61,7 +61,7 @@ func newSchemaWrap4Owner(kvStorage tidbkv.Storage, startTs model.Ts, config *con
 		schemaSnapshot: schemaSnap,
 		filter:         f,
 		config:         config,
-		ddlHandledTs:   startTs,
+		ddlHandledTs:   startTs - 1,
```

Comment:
Can we keep using `startTs` in this function, L44, L52 and L 64, but change the caller to pass `checkpointTs - 1`, from `cdc/owner/changefeed.go`, and add comment the `startTs` is `[startTs, ...)` in this function

Reply:
sound reasonable